docs(dwc): add doc on persistent volume MTA-5885 (#4978)
scaleway/docs-content: pages/data-lab/concepts.mdx

  - managed-services
---

> ## Apache Spark Cluster 

Reviewer: ```suggestion
## Apache Spark cluster
```

> An Apache Spark cluster is an orchestrated set of machines over which the distributed/Big data calculus is going to be processed. In the case of this project, the Apache Spark cluster is a Kubernetes cluster, upon which Apache Spark has been installed in every pod deployed. For more details, check out the [Apache Spark documentation](https://spark.apache.org/documentation.html).

Reviewer 2: ```suggestion
An Apache Spark cluster is an orchestrated set of machines over which distributed/Big data calculus is processed. In the case of Scaleway Data Lab, the Apache Spark cluster is a Kubernetes cluster, with Apache Spark installed in each pod. For more details, check out the [Apache Spark documentation](https://spark.apache.org/documentation.html).
```
